pixi.toml

```diff
@@ -0,0 +1,168 @@
+[project]
+name = "datajudge"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "win-64"]
+
+[tasks]
+postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
+
+[dependencies]
+python = ">=3.8"
+scipy = "*"
+sqlalchemy = ">=1.4,<3"
+flit-core = "*"
+flit = "*"
+colorama = "*"
+pytest = "*"
+
+[feature.docs.dependencies]
+make = "*"
+numpydoc = "*"
+sphinx = "*"
+sphinx_rtd_theme = "*"
+sphinxcontrib-apidoc = "*"
+sphinx-autodoc-typehints = "*"
+[feature.docs.tasks]
+docs = "cd docs && make html "
+readthedocs = "rm -rf $READTHEDOCS_OUTPUT/html && cp -r docs/build/html $READTHEDOCS_OUTPUT/html"
+
+[feature.py38.dependencies]
+python = "3.8.*"
+[feature.py39.dependencies]
+python = "3.9.*"
+[feature.py310.dependencies]
+python = "3.10.*"
+[feature.py311.dependencies]
+python = "3.11.*"
+[feature.py312.dependencies]
+python = "3.12.*"
+
+[feature.unit.tasks]
+test = "pytest tests/unit"
+coverage = "pytest tests/unit --cov=datajudge --cov-report=xml --cov-append"
+
+[feature.mssql.dependencies]
+pyodbc = "*"
+freetds = "*"
+[feature.mssql.tasks]
+test = "pytest tests/integration --backend=mssql"
+test_freetds = "pytest tests/integration --backend=mssql-freetds"
+coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=mssql"
+
+[feature.postgres.dependencies]
+psycopg2 = "*"
+[feature.postgres.tasks]
+test = "pytest tests/integration --backend=postgres"
+coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=postgres"
+
+[feature.db2.dependencies]
+ibm_db = "*"
+ibm_db_sa = "*"
+[feature.db2.tasks]
+test = "pytest tests/integration --backend=db2"
+coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=db2"
+
+[feature.snowflake.dependencies]
+snowflake-sqlalchemy = "*"
+[feature.snowflake.tasks]
+test = "pytest tests/integration --backend=snowflake"
+coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=snowflake"
+
+[feature.bigquery.dependencies]
+sqlalchemy-bigquery = "*"
+[feature.bigquery.tasks]
+test = "pytest tests/integration --backend=bigquery -v -n=16"
+coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=bigquery -v -n=16"
+
+[feature.impala.dependencies]
+impyla = "*"
+[feature.impala.tasks]
+test = "pytest tests/integration --backend=impala"
+coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=impala"
+
+[feature.sa1.dependencies]
+sqlalchemy = "1.*"
+
+[feature.sa2.dependencies]
+sqlalchemy = "2.*"
+
+[feature.test.dependencies]
+pytest = "*"
```

Comment:
`pytest` is a dependency in the `dependencies` main block.

Reply:
Good catch, removed it! 
aaa5cbf